CriticalMapsKit/Sources/AppFeature/RequestTimerCore.swift

```diff
@@ -2,7 +2,7 @@ import ComposableArchitecture
 import Foundation
 
 public struct RequestTimer: ReducerProtocol {
-  public init(timerInterval: Int = 12) {
+  public init(timerInterval: Int = 15) {
```

Comment:
```suggestion
  public init(timerInterval: Int = 30) {
```

Reply:
we really need to lower the load on the API. as far as i understand this is the interval for sending the own location. can we set the interval to 30 seconds instead?